include/aktualizr-lite/cli/cli.h

```diff
@@ -55,14 +55,30 @@ enum class PullMode {
   None,
 };
 
+/**
+ * The TUF check mode to be used.
+ */
+enum class CheckMode {
+  /**
+   * Default mode, do update TUF metadata from remote of local source.
+   */
+  Update = 0,
+  /**
+   * Do not update TUF roles: use currently stored metadata.
+   */
+  Current,
+};
+
 StatusCode CheckIn(AkliteClient &client, const LocalUpdateSource *local_update_source = nullptr);
 
 StatusCode Pull(AkliteClientExt &client, int version = -1, const std::string &target_name = "",
-                bool force_downgrade = true, const LocalUpdateSource *local_update_source = nullptr);
+                bool force_downgrade = true, const LocalUpdateSource *local_update_source = nullptr,
+                CheckMode check_mode = CheckMode::Update);
 
 StatusCode Install(AkliteClientExt &client, int version = -1, const std::string &target_name = "",
                    InstallMode install_mode = InstallMode::All, bool force_downgrade = true,
-                   const LocalUpdateSource *local_update_source = nullptr, PullMode pull_mode = PullMode::All);
+                   const LocalUpdateSource *local_update_source = nullptr, PullMode pull_mode = PullMode::All,
+                   CheckMode check_mode = CheckMode::Update);
```

Comment:
If it is set to `CheckMode::Update` by default then I think `cli_install` and `install` functions in main.cc require adjustment so `CheckMode::Current` mode is applied in the `cli_install` case?

Reply:
It's been addressed.